helm/aws-load-balancer-controller/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: aws-load-balancer-controller
 description: AWS Load Balancer Controller Helm chart for Kubernetes
-version: 1.5.1
+version: 1.5.2
```

Comment:
lets keep the chart version at 1.5.1, we will update it when publishing the chart.

Reply:
done